shed/tests/test_streams.py

```diff
@@ -1043,10 +1062,10 @@ def test_curate_streams():
     doc4_curated2 = s.curate_streams(doc4_curated, True)
     doc5_curated2 = s.curate_streams(doc5_curated, True)
 
-    assert doc1_curated == ('start', (None, ))
-    assert doc1_curated2 == ('start', None, )
+    assert doc1_curated == ('start', (None,))
+    assert doc1_curated2 == ('start', None,)
```

Comment:
but it already is one with the first comma, unless it's meant to be a `tuple` within a `tuple`? (`('start', (None,))`)